### PR TITLE
Better seeds

### PR DESCRIPTION
**Pull request by kalkov**
#### Summary
Few manual steps were needed to have a properly working project - adding contract and adjusting token decimals. Should not be the case anymore.

[//]: # (#### Related PRs)

[//]: # (#### Additional deploy notes)

[//]: # (#### Screenshots)

### CODE REVIEW

File: priv/repo/seeds.exs [hunk 1]
```diff
@@ -164,39 +167,39 @@ end
...
+|> Enum.each(insert_on_conflict_nothing)
 
-user =
-  %User{
-    email: "john.d@santiment.net",
-    username: "John Dow",
-    salt: "LgcwR3e98PR/gvgV7Ph1+ZXnw4yhTz25k08QLi/39qdCt/V0XOGlJRiL938NtJk0"
-  }
-  |> Repo.insert!()
+defmodule InsertUser do
```
Comment (tspenov) [inline priv/repo/seeds.exs]: I think we can take it a step further and extract these module into own files and in seeds file we have only the raw data and calls to`InsertUser.run, etc...` functions. Also for the `Projects`, `InsertExchangeAddresses`, `Infrastructure` etc... What do you think ?

Comment (tspenov) [reply priv/repo/seeds.exs]: Also all of these `.run` functions to be made to handle a list of tuples of data.

Comment (tspenov) [reply priv/repo/seeds.exs]: But of course these changes are still an improvement so feel free to skip if you think it is too out of scope.

Comment (IvanIvanoff) [reply priv/repo/seeds.exs]: https://github.com/santiment/sanbase2/blob/master/priv/timescale_repo/seeds.exs

Comment (kalkov) [reply priv/repo/seeds.exs]: @tspenov Good suggestion, will step further but in separate PR. The idea behind this PR was to quickly fix the seeds so I can continue working.

**Pull request merged**